modules/models/src/main/java/org/eclipse/xpanse/modules/models/enums/Csp.java

```diff
@@ -18,7 +18,8 @@ public enum Csp {
     AZURE("azure"),
     ALI("alibaba"),
     HUAWEI("huawei"),
-    OPENSTACK("openstack");
+    OPENSTACK("openstack"),
+    FLEXIBLE("flexible");
```

Comment:
FLEXIBLEENGINE("flexibleEngine") I am going to do the same change on the xpanse-ui.

Reply:
Better as FLEXIBLE_ENGINE.. with underscore separated.

Reply:
yes, I believe `FLEXIBLE_ENGINE` is better.

Reply:
Ok. I renamed it to FLEXIBLE_ENGINE